src/components/Dropdown/DropdownInput.tsx

```diff
@@ -37,15 +37,19 @@ export function DropdownInput(props: {
 
   const handleChange = useCallback((e: ChangeEvent<HTMLInputElement>) => {
     toggleDropdown(true);
+    onChange?.(e.target.value);
     updateFocusedItem(-1, e.target.value);
     setLastTypedOrSubmittedValue(e.target.value);
-    onChange?.(e.target.value);
   }, [onChange, setLastTypedOrSubmittedValue, toggleDropdown, updateFocusedItem]);
 
   const handleKeyDown = useCallback((e: KeyboardEvent<HTMLInputElement>) => {
     if (e.key === 'Enter' && (!submitCriteria || submitCriteria(focusedIndex))) {
+      let newValue;
+      if (focusedItemData?.filter && typeof focusedItemData?.filter === 'object' && typeof focusedItemData?.filter['value'] === 'string') {
```

Comment:
Why does DropdownInput need to know the contents of `focusedItemData`?

Reply:
We changed it so that arrowing/tabbing doesn't change the value in the input anymore, so pressing enter sets the value in the input to the focused value

Reply:
DropdownInput shouldn't know what your focusedItemData is. Could add some kind of `defaultItem` or `defaultValue` prop to Dropdown? This value/item would be used for the `-1` index

Reply:
Alternatively instead of a new prop, is you have `alwaysSelectOption` true the -1 index would just point to the first element

Reply:
What exactly is default thing be for? To clarify, this is only setting the value in the input when the user presses enter; arrowing down would keep the value the same which is why I had to change the `setValue` in `updateFocusedItem`

Reply:
I was just trying to brainstorm solutions. Regardless how you do it, it's important DropdownInput doesn't access `focusedItemData`.

Reply:
gotcha. Would it be ok to add a new param to toggleDropdown and tell it to set the value?

Reply:
Are you saying you would pass a `setValue` callback down to `handleToggleDropdown`? That's not ideal but probably ok for now. If you wanted to do it that way couldn't you call `setFilterQuery` instead of adding a new param?

Reply:
No I meant the `toggleDropdown` method in `Dropdown`. If I had a param that was like `selected` I could probably call `setValue` there- and if the user clicked out instead of clicking a filter/entering then `selected` would be `false` and wouldn't change the input value

Reply:
Sure I'm not exactly sure what you're describing but give it a go. If you want I can get on zoom